test/kitchen/test/integration/container-collection-test/rspec/container-collection-test_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+shared_examples_for 'a running Process Agent' do
+    it 'is running' do
+        if os == :windows
+            expect(is_process_running?("process-agent.exe")).to be_truthy
+        else
+            expect(is_process_running?("process-agent")).to be_truthy
+        end
+    end
+end
+
+describe 'a Process Agent with Container Collection enabled' do
+    it_behaves_like 'a running Process Agent'
+    it 'has container collection enabled' do
+        conf = get_runtime_config()
+        expect(conf).to have_key("container_collection")
+        expect(conf["container_collection"]).to have_key("enabled")
+        expect(conf["container_collection"]["enabled"]).to be_truthy
+    end
+    it 'is running the container check' do
+        expect(check_enabled?("container")).to be_truthy
```

Comment:
what do you think of checking for `rtcontainer` and `rtprocess` (in `process-collection-test`) as well?

Reply:
Ah good point, I will do that